### PR TITLE
[17.0][MIG] hr_employee_phone_extension : Migration to 17.0

### PR DESCRIPTION
**Pull request by manishkumarbohra**
Module Name : Employee Phone Extension(hr_employee_phone_extension)

Migration to 17.0

For Issue #1291

### CODE REVIEW

File: hr_employee_phone_extension/readme/CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+- Sandy Carter \<<sandy.carter@savoirfairelinux.com>\>
+- El Hadji Dem \<<elhadji.dem@savoirfairelinux.com>\>
+- Vincent Renaville \<<vincent.renaville@camptocamp.com>\>
+- Charbel Jacquin \<<charbel.jacquin@camptocamp.com>\>
+- Denis Leemann \<<denis.leemann@camptocamp.com>\>
+- Serpent Consulting Services Pvt. Ltd.\<<support@serpentcs.com>\>
+- Anand Kansagra \<<kansagraanand@hotmail.com>\>
+- Saran Lim. \<<saranl@ecosoft.co.th>\>
+- aphon61bank \<<aphon61bank@gmail.com>\>
+- Manish Bohra \<<manishkumarbohra@outlook.com>\>
```
Comment (Saran440) [inline hr_employee_phone_extension/readme/CONTRIBUTORS.md:10]: Why you change file in readme folder from `.rst` to `.md`?

File: hr_employee_phone_extension/readme/INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+To install this module, you need to:
```
Comment (Saran440) [inline hr_employee_phone_extension/readme/INSTALL.md:1]: This file you can delete it because this step to install generic.

**Review by Saran440 — COMMENTED** [review]
Hi, Thank you for your contribution.

Code Review.

**Comment by victoralmau** [comment]
Please, cherry-pick https://github.com/OCA/hr/pull/1353 to commit history before migration commit.

**Pull request closed without merging**